app/views/groups/manage/dashboard.scala.html

```diff
@@ -24,22 +24,59 @@
 @this(
     layout: main_layout, p: p, h1: h1, h2: h2, h3: h3, a: a,
     table: table, textbox: textbox, input_select: input_select,
-    a_button: link_as_button, summary_list: summary_list, govukTabs: GovukTabs,
-    selectable_clients_table_form: selectable_clients_table_form
+    a_button: link_as_button,summary_list: summary_list, govukTabs: GovukTabs,
+    selectable_clients_table_form: selectable_clients_table_form,
+        formWithCSRF: FormWithCSRF,
+        submit_button_group: submit_button_group,
+        summaryErrors: SummaryErrors
 )
 
 @(
     summaries: (Seq[GroupSummary], Seq[DisplayClient]),
-    form: Form[AddClientsToGroup],
+    addClientsToGroupForm: Form[AddClientsToGroup],
+        filterGroupSummaryForm: Form[String],
     hiddenClientsExist: Option[Boolean],
     showUnassignedClients: Boolean = false
 )(implicit request: Request[_], msgs: Messages, appConfig:
 AppConfig)
 
+
 @groupSummaries = {
     @h2("group.manage.tabs.1.label")
     <div class="govuk-!-width-one-third"></div>
-    @if(!summaries._1.isEmpty){
+
+
+
+@formWithCSRF(routes.ManageGroupController.submitFilterByGroupName){
+
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+        @textbox(
+            field = filterGroupSummaryForm("search"),
+            label = "group-name-filter.text.label",
+            labelClasses = "govuk-label--s",
+            classes = "govuk-input--width-20"
+        )
+        </div>
+        <div class="govuk-grid-column-one-third" id="filter-buttons-wrapper">
+            @submit_button_group()
+        </div>
+        <div class="govuk-grid-column-one-third">
+            @input_select(
+                field = filterGroupSummaryForm("sort"),
+                label = msgs("Sort by"),
+                options = Seq(("default" ,"Default (A-Z)"), ("reverse", "Z-A")),
+                firstItem =  Seq.empty[SelectItem]
```

Comment:
you shouldn't need this line if you've made it the default in the component

Reply:
which line please? then I'll remove, thx

Reply:
line 70, sorry I thought I selected it

Reply:
ok removed now please check